app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -21,6 +21,7 @@ import org.simple.clinic.facility.FacilityRepository
 import org.simple.clinic.medicalhistory.MedicalHistoryRepository
```

Comment:
I can see that this commit is doing more than adding a test.
Can you separate out the changes to the query in a different commit?